sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -42,6 +43,10 @@ const Lottery = (props: { listing: Listing }) => {
   const { listingsService, profile } = useContext(AuthContext)
   const { data } = useFlaggedApplicationsMeta(listing?.id)
   const duplicatesExist = data?.totalPendingCount > 0
+  const shouldExpireData =
+    !profile?.userRoles.isAdmin && listing?.status === ListingsStatusEnum.closed
+  const expiryDate = dayjs(listing?.closedAt).add(45, "day")
```

Comment:
Should this 45 come from an env variable if it's an env variable on the backend?

Reply:
Definitely, good catch

Reply:
Alternatively, should the the listing dto send this to the frontend so we only have 1 env var on the backend instead of us having to set the same var in multiple places?